ocfweb/account/register.py

```diff
@@ -107,6 +109,23 @@ def request_account(request):
         },
     )
 
+def recommend(request):
+    real_name = request.GET.get('real_name', '')
+    first_name, last_name = real_name.split()
+    rec_lst = recommender.recommend(first_name, last_name, 3)
+    rec_dict = {}
+    for i in range(len(rec_lst)):
+        rec_dict[str(i)] = rec_lst[i] # convert i to string because JS dicts need string keys
+    return JsonResponse(rec_dict)
+
+def validate(request):
+    real_name = request.GET.get('real_name', '')
+    username = request.GET.get('username', '')
+    try:
+        validate_username(username, real_name)
+        return JsonResponse({'is_valid': True})
+    except:
+        return JsonResponse({'is_valid': False})
```

Comment:
Instead of returning a boolean, can we return a string, i.e. one of `"available"`, `"taken"`, or `"invalid"`? That way we can give more specific feedback on the form.